.gitignore

```diff
@@ -8,3 +8,6 @@ e-commerce-frontend/.vscode/
 shop4All-backend/.idea/workspace.xml
 shop4All-backend/.idea/workspace.xml
 shop4All-backend/.idea/workspace.xml
+shop4All-backend/logs/myapp.log.2024-08-26.0.gz
+shop4All-backend/logs/myapp.log
```

Comment:
Is it possible to only leave "shop4All-backend/logs"? Normally, this means that everything from the logs folder will be ignored by git.